flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/factory/MySqlDataSourceFactory.java

```diff
@@ -243,6 +271,14 @@ private static List<String> getTableList(MySqlSourceConfig sourceConfig, Selecto
                 .collect(Collectors.toList());
     }
 
+    private static List<ObjectPath> getChunkKeyColumnTableList(
+            MySqlSourceConfig sourceConfig, Selectors selectors) {
+        return MySqlSchemaUtils.listTables(sourceConfig, null).stream()
```

Comment:
Could we invoke `listTables` once and filter tables in the same list for each selector?